pkg/cmd/runsSubmitLocal.go

```diff
@@ -41,6 +41,10 @@ func init() {
 		"https://repo.maven.apache.org/maven2",
 		"the url of the remote maven where galasa bundles can be loaded from. "+
 			"Defaults to maven central.")
+	
+
+	runsSubmitLocalCmd.Flags().StringVar(&runsSubmitLocalCmdParams.LocalMaven, "localMaven", "",
+		"The url of a local maven repository are where galasa bundles can be loaded from on your local file system. Defaults to your home .m2/repository file.")
```

Comment:
Im in favour of more help here rather than less. In the readme you had more, can you put that detail here. An example or two ? Maybe a windows example ?